shell/conda.fish

```diff
@@ -190,14 +190,13 @@ function activate --description 'Activate a conda environment.'
         else
           set -gx CONDA_DEFAULT_ENV $argv[1]
         end
-
         # Always store the full prefix path as CONDA_PREFIX
         set -gx CONDA_PREFIX (echo $PATH[1] | sed 's|/bin$||g')
 
         # check if there are any *.fish scripts in activate.d
         set -l activate_d $CONDA_PREFIX/etc/conda/activate.d
```

Comment:
Is this the correct place? In the root env? Or does every env have its own `etc/conda/(de)activate.d` folders? Should we take that possibility into account?